tester_app/client/app/bundles/HelloWorld/containers/HelloWorld.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes } from 'react';
+import HelloWorldWidget from '../components/HelloWorldWidget';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import Immutable from 'immutable';
+import * as helloWorldActionCreators from '../actions/helloWorldActionCreators';
+
+function select(state) {
+  // Which part of the Redux global state does our component want to receive as props?
+  return { $$helloWorldStore: state.$$helloWorldStore };
+}
+
+class HelloWorld extends React.Component {
+  static propTypes = {
+    dispatch: PropTypes.func.isRequired,
+    $$helloWorldStore: PropTypes.instanceOf(Immutable.Map).isRequired,
+  }
+
+  render() {
+    const { dispatch, $$helloWorldStore } = this.props;
+    const actions = bindActionCreators(helloWorldActionCreators, dispatch);
+    return (
+      <HelloWorldWidget $$helloWorldStore={$$helloWorldStore} actions={actions} />
```

Comment:
Yep, it's right. When to use it is really matter of taste. I made up it while was dealing with component, where were around 5 attributes with 100% repetition of key / value ([link](http://alexfedoseev.com/post/61/react-component-props-definition-with-es2015-spread-operator)). My DRY alarm system works!